packages/svelte-docgen/src/parser/events.test.ts

```diff
@@ -40,296 +40,15 @@ describe("events", () => {
-			                          "kind": "literal",
-			                          "subkind": "boolean",
-			                          "value": true,
-			                        },
-			                      ],
-			                    },
-			                  },
-			                  "cancelable" => {
-			                    "isOptional": true,
-			                    "isReadonly": false,
-			                    "type": {
-			                      "kind": "union",
-			                      "nonNullable": {
-			                        "kind": "boolean",
-			                      },
-			                      "types": [
-			                        {
-			                          "kind": "undefined",
-			                        },
-			                        {
-			                          "kind": "literal",
-			                          "subkind": "boolean",
-			                          "value": false,
-			                        },
-			                        {
-			                          "kind": "literal",
-			                          "subkind": "boolean",
-			                          "value": true,
-			                        },
-			                      ],
-			                    },
-			                  },
-			                  "composed" => {
-			                    "isOptional": true,
-			                    "isReadonly": false,
-			                    "type": {
-			                      "kind": "union",
-			                      "nonNullable": {
-			                        "kind": "boolean",
-			                      },
-			                      "types": [
-			                        {
-			                          "kind": "undefined",
-			                        },
-			                        {
-			                          "kind": "literal",
-			                          "subkind": "boolean",
-			                          "value": false,
-			                        },
-			                        {
-			                          "kind": "literal",
-			                          "subkind": "boolean",
-			                          "value": true,
-			                        },
-			                      ],
-			                    },
-			                  },
-			                },
-			                "sources": Set {
-			                  /node_modules/.pnpm/typescript@<semver>/node_modules/typescript/lib/lib.dom.d.ts,
-			                },
-			              },
-			            ],
-			          },
-			        },
-			      ],
-			    ],
-			    "kind": "constructible",
-			    "name": "CustomEvent",
-			    "sources": Set {
-			      /node_modules/.pnpm/typescript@<semver>/node_modules/typescript/lib/lib.dom.d.ts,
-			    },
-			  },
-			  "on:increment" => {
-			    "constructors": "self",
-			    "kind": "constructible",
-			    "name": "CustomEvent",
-			    "sources": Set {
-			      /node_modules/.pnpm/typescript@<semver>/node_modules/typescript/lib/lib.dom.d.ts,
-			    },
-			  },
+			  "on:decrement" => "CustomEvent<number>",
```

Comment:
**NICE! 🎉** 
I think you solved one of the limitations I've found in #3. I thought it's a `svelte2tsx` issue.